lib/hanami/generators/application/app/templates/application.html.slim.tt

```diff
@@ -0,0 +1,8 @@
+doctype html
+html
+  head
+    title
+      | <%= config[:classified_app_name] %>
+    - = favicon
```

Comment:
@aderyabin This dash here causes an exception. Please have a look at the attached screenshot. Can you confirm the problem?

![screen shot 2016-02-15 at 18 13 56](https://cloud.githubusercontent.com/assets/5089/13055501/02812f4a-d410-11e5-8b00-2ad05e8c3e8b.png)